types/node-fetch/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "form-data": "^2.3.3"
+        "form-data": "2.5.0"
```

Comment:
If you take this suggestion perhaps change the PR description to something like:

```
fix(node-fetch): restore form-data dependency to ^3.0.0

We recently restored the deleted node-fetch types. We lowered
the dependency on form-data from `^3.0.0` to `^2.3.3` since that's
what the dev dependency in `node-fetch@2` itself had but that older
version didn't ship its own types.

This PR restores the dependency to the same constraint it had when
we previously published `@types/node-fetch`. While it seems a bit
suspect to depend on a newer major version than `node-fetch` does,
the only change in `form-data` v3 is dropping `node@4` support, and
it's just a dev dependency upstream.
```

Reply:
Your call. Technically that's breaking dep for `node-fetch`, requiring 2.3, isn't it? We are crushing copies :), the best would be if `@types/form-data` was whitelisted on EOL.

Reply:
Not sure what EOL is...